server/executor/trigger/grpc.go

```diff
@@ -49,15 +57,22 @@ func (te *grpcTriggerer) Trigger(_ context.Context, test model.Test, tid trace.T
 	if trigger.GRPC == nil {
 		return response, fmt.Errorf("no settings provided for GRPC triggerer")
 	}
+
+	ctx, span := te.tracer.Start(context.Background(), "Tracetest Trigger")
```

Comment:
I think it would be better to have the span and ctx set outside the trigger, so each trigger doesn't have to copy/paste this code